docs/pages/admin-guides/teleport-policy/integrations/gitlab.mdx

```diff
@@ -113,14 +113,14 @@ You can also run queries to fetch specific information from the Access Graph, su
 The following query fetches all projects accessible to a user <Var name="user" />:
 
 ```sql
-SELECT * FROM access_path WHERE "identity" = '<Var name="user" />' AND source='GITLAB'
+SELECT * FROM access_path WHERE "identity" = '<Var name="user" />' AND source='Gitlab'
```

Comment:
The product is called "GitLab". Is "Gitlab" a hardcoded name within Teleport?

Reply:
Yes in access graph

Reply:
👍 For later, is there a way to alias "Gitlab" as "GitLab" so customers can use the correct capitalization alongside the incorrect one?

Reply:
Each integration has different casing, we should probably make that field case insensitive (for backward compatibility) and update our docs with lowercase examples to make it simpler for everyone.